### ISSUE TITLE
Accessibility - Missing image ALT tags and wrong headings order

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#3291]
Branch: develop
Expected Behavior: "All non-text content that is presented to the user has a text alternative that serves the equivalent purpose." (WCAG 2.0 Guideline 1.1.1.)

WAVE checker is also complaining about incorrectly ordered headings (WCAG 2.0 Guideline 2.4.6)

Current Behavior: Current ka-lite server **homepage** has a CC logo in the footer and the linked "Learn" image without the appropriate ALT tags. 

"A free world-class education for anyone anywhere." title should be H1 instead of H2

(image is from the master branch, so slightly different from develop)
![homepage-wave-errors](https://cloud.githubusercontent.com/assets/1457929/6646684/d50910e4-c9c7-11e4-83fc-d213240614d7.png)

**Comment by MCGallaspy** [comment]
Looks like this is resolved by #3292, @radinamatic?

**Comment by radinamatic** [comment]
This is probably the very first issue I reported here... :P 
And, yes, it's been resolved!
